### ISSUE TITLE
Ask for confirmation before deleting *any* quantity of items

### ISSUE THREAD

**Issue by BloodyIron** [nextcloud/server#47931]
It's very easy to hit the Delete functions in the webGUI in the Files section without meaning it. I really don't want to have to restore files every single time I accidentally hit that button as that's time costly. I really would prefer we just have a dialogue of "Are you sure?" with Yes/No, or equivalent, pop up, every time, regardless of how many files are selected (singular, multiple, whatever).

Can we please have this added? This is how file handling workflows and UX works commonly across the majority of user GUIs that deal with file handling.

**Comment by joshtrichards** [comment]
We do: #46733 (upcoming v30) & #43231 (v29)

Also, some further refinement discussed here: #46732

**Comment by BloodyIron** [comment]
No, I'm talking about even if the trashbin is enabled. I was told to create a new request in another thread. PLEASE RE OPEN THIS. Disabling the trashbin should not be required for this dialogue! The request is in ALL scenarios.

**Comment by joshtrichards** [comment]
#43231 does not require the trashbin to be enabled. It does, however, require >5 items. I'll re-open this for eliminating the item requirement.

**Comment by BloodyIron** [comment]
> #43231 does not require the trashbin to be enabled. It does, however, require >5 items. I'll re-open this for eliminating the item requirement.
> 
> EDIT: Adjusted the title to make it clearer

Indeed! Yes this is now in-line with what I had in mind, thanks for re-opening and whatnot :)
